crates/red_knot_python_semantic/resources/mdtest/type_properties/is_subtype_of.md

```diff
@@ -30,9 +28,6 @@ static_assert(not is_subtype_of(int, bool))
 static_assert(not is_subtype_of(int, str))
 static_assert(not is_subtype_of(object, int))
 
-static_assert(not is_subtype_of(int, float))
```

Comment:
Not sure if we have to, but we could potentially keep those tests using something like
```py
type BareFloat = knot_extensions.TypeOf[1.0]
```

Reply:
Done!

Reply:
(Also, it's really cool that we can do this!)